<ui version="4.0" >
 <class>CSettingsDlgClass</class>
 <widget class="QDialog" name="CSettingsDlgClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CSettingsDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item rowspan="3" row="0" column="0" >
    <widget class="QListWidget" name="contentsWidget" >
     <property name="maximumSize" >
      <size>
       <width>128</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="movement" >
      <enum>QListView::Snap</enum>
     </property>
     <property name="spacing" >
      <number>12</number>
     </property>
     <property name="viewMode" >
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QStackedWidget" name="pagesWidget" >
     <widget class="QWidget" name="page" />
     <widget class="QWidget" name="page_2" />
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
